src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -34,13 +35,13 @@
     <Compile Include="System\IO\Pipelines\ThrowHelper.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
-    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetCoreApp21.cs" />
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.netcoreapp.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.3'">
-    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard13.cs" />
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.netstandard1.3.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netstandard'">
-    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard.cs" />
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.netstandard2.0.cs" />
```

Comment:
Nit: The configuration name is netstandard, so the suffix should be just `netstandard.cs`